### PR TITLE
Remove `parlai/mturk/`

### PR DESCRIPTION
**Pull request by EricMichaelSmith**
**Patch description**
- Removes the code in `parlai/mturk/` and leaves a README of how to access it via tags
- Changes/removes references to `parlai/mturk/`
- Removes the now-obsolete `mturk` CI check

Note that the MTurk tutorial will be rewritten in an upcoming PR to reflect the new `parlai.crowdsourcing` code

**Testing steps**
crowdsourcing CI checks

### CODE REVIEW

**Review by stephenroller — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -165,7 +165,7 @@ commands:
             command: |
               cd ..
               git clone git@github.com:facebookresearch/Mephisto.git Mephisto
-              cd Mephisto; git checkout v0.3.1.1 -b stable
+              cd Mephisto; git checkout 2e8e131a8c27ccabeccc4af894777d1b2ca449fe -b stable
```
Comment (stephenroller) [inline .circleci/config.yml:168]: i still don't like these.

we should be pip installing here, not checking out code

File: docs/source/tutorial_mturk.md [hunk 2]
```diff
@@ -10,15 +10,9 @@ instructions for hosting a ParlAI-Chat style task can be found
 and regular collection tasks can be found
 [here](https://github.com/facebookresearch/Mephisto/tree/master/examples/static_react_task).
 
-We will be moving some of our common tasks out of the Mephisto repo into
```
Comment (stephenroller) [inline docs/source/tutorial_mturk.md:13]: change the box to be yellow or red now, plz

File: parlai/mturk/README.md [hunk 3]
```diff
@@ -1,7 +1,167 @@
...
+If you just need to read the code, for reference, you may browse it
+[here](https://github.com/facebookresearch/ParlAI/tree/mturk_archive/parlai/mturk/tasks/dealnodeal).
+
+
+### tasks/image_chat/
+
+The current Mephisto-based version of the human+model image-chat task can be found in [`parlai.crowdsourcing.tasks.model_chat`](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing/tasks/model_chat). The obsolete pre-Mephisto version of this task can be restored by switching to the `v0.10.0` tag of ParlAI:
+
+```bash
+git checkout v0.10.0
```
Comment (stephenroller) [inline parlai/mturk/README.md:51]: i reckon we could make them all mturk_archive

File: parlai/mturk/README.md [hunk 4]
```diff
@@ -1,7 +1,167 @@
...
 
-- **core**: contains all the files that are used to make the underlying MTurk interaction work properly
-- **tasks**: contains a number of tasks that have been launched on MTurk to collect various datasets (shared for reproducibility), as well as a few tasks that exist as examples of possible MTurk functionality
-- **scripts/**: contains various helpful review and cleanup scripts to manage tasks from the command line (especially after mistakes or major failures).
-- **webapp**: contains server and build files for the [Beta] ParlAI-MTurk frontend, which is currently under heavy development and may undergo many changes
-- **run_data/**: contains all of the cross-run state in sqlite databases (default `pmt_data.db` and `pmt_sbdata.db`), as well as any world data saved by the default behavior of `MTurkDataWorld` (in `live` and `sandbox` folders, following the behavior specified in `MTurkDataHandler`).
+Instructions on how to find all old `parlai/mturk/` code can be found in the sections below.
```
Comment (stephenroller) [inline parlai/mturk/README.md:3]: add a `-----------` below this

File: .circleci/config.yml (hunk 1 above)
Comment (EricMichaelSmith) [reply .circleci/config.yml:168]: The hardcoded commit SHA here will be changed to a clean tag name before merging in the PR (I left a note in the PR description about this). But I'd definitely support `pip install`ing a specific version of Mephisto - @JackUrb do you know if that is supported?

File: docs/source/tutorial_mturk.md (hunk 2 above)
Comment (EricMichaelSmith) [reply docs/source/tutorial_mturk.md:13]: Makes sense, changing to yellow warning

File: parlai/mturk/README.md (hunk 3 above)
Comment (EricMichaelSmith) [reply parlai/mturk/README.md:51]: Hmm, that'd be hard - for instance, `convai2_model_eval` was removed *before* the `mturk_archive` tag, and I added a lot of new features to the pre-Mephisto turn annotations task a few days *after* the `mturk_archive` tag, so just switching to that tag wouldn't capture the final version of each of the tasks simultaneously

File: parlai/mturk/README.md (hunk 4 above)
Comment (EricMichaelSmith) [reply parlai/mturk/README.md:3]: Yes, added

File: .circleci/config.yml (hunk 1 above)
Comment (JackUrb) [reply .circleci/config.yml:168]: We're close to the pip install, just a few rough edges I haven't figured out yet. Should be there by the end of the month.

**Review by JackUrb — APPROVED** [review]
Moving on has been a long time coming - thanks for getting this over the finish line!

**Pull request merged**